include/wabt/c-writer.h

```diff
@@ -18,15 +18,17 @@
 #define WABT_C_WRITER_H_
 
 #include "wabt/common.h"
+#include "wabt/feature.h"
 
 namespace wabt {
 
 struct Module;
 class Stream;
 
 struct WriteCOptions {
+  WriteCOptions(const Features& features) : features(features) {}
   std::string_view module_name;
-  bool no_sandbox;
+  const Features& features;
```

Comment:
Also maybe make a copy here? It is unclear from the outside that caller is responsible for keeping the instance alive. Alternatively pass a pointer to c-tor to make it clear(er) that the caller is owning this object.